Engines/Wine/Engine/Implementation/script.js

```diff
@@ -347,13 +347,9 @@ class WineEngine {
         this.install(subCategory, version);
 
         var wineBinary = this.getLocalDirectory(subCategory, version) + "/bin/wine";
-        var StringArray = Java.type('java.lang.String[]');
-        var command = new StringArray(2 + args.length);
-        command[0] = wineBinary;
-        command[1] = executable;
-        java.lang.System.arraycopy(args, 0, command, 2, args.length);
+        var command = [wineBinary, executable].concat(args);
         var ProcessBuilderClass = Java.type('java.lang.ProcessBuilder');
-        var processBuilder = new ProcessBuilderClass(command);
+        var processBuilder = new ProcessBuilderClass(Java.to(command, Java.type("java.lang.String[]")));
```

Comment:
Is there a rule for `"java.lang.String[]"` vs `'java.lang.String[]'`. I think we should be consistent, see e.g. `'java.lang.ProcessBuilder'`.

Reply:
What do you mean by "java.lang.String[]" vs 'java.lang.String[]'?

Reply:
single vs double quotes

Reply:
No there is not, I simply copied the `Java.type(...)` command from somewhere else^^

Reply:
Nevertheless I think we should always use the same. Lets say single quotes, ok?